databricks/koalas/tests/test_frame_plot.py

```diff
@@ -335,10 +335,26 @@ def _test_hist_plot(pdf, kdf):
         kdf.columns = columns
         _test_hist_plot(pdf, kdf)
 
+    def test_kde_plot(self):
+        pdf = self.pdf1
+        kdf = self.kdf1
+
+        ax1 = pdf.plot.kde(bw_method=3)
+        ax2 = kdf.plot.kde(bw_method=3)
+        self.compare_plots(ax1, ax2)
+
+        ax1 = pdf.plot(kind='kde', bw_method=3)
+        ax2 = kdf.plot(kind='kde', bw_method=3)
+        self.compare_plots(ax1, ax2)
+
+        ax1 = pdf.plot.kde(bw_method=3, ind=[1, 2, 3])
+        ax2 = kdf.plot.kde(bw_method=3, ind=[1, 2, 3])
+        self.compare_plots(ax1, ax2)
```

Comment:
Can you comment out this test @itholic with the explanation above? Let's merge this one first.

Reply:
We should fix whole test cases except only some test cases in plot - it seems it was wrong at the very first place.